docs/sources/reference/components/loki.source.awsfirehose.md

```diff
@@ -64,6 +64,19 @@ loki.source.awsfirehose "LABEL" {
 The component will start an HTTP server on the configured port and address with the following endpoints:
 
 - `/awsfirehose/api/v1/push` - accepting `POST` requests compatible with [AWS Firehose HTTP Specifications](https://docs.aws.amazon.com/firehose/latest/dev/httpdeliveryrequestresponse.html).
+- `static_configs_labels` - query parameter can be used to set extra static labels. It should contain base64 encoded YAML with labels values. For example:
```

Comment:
```suggestion

You can use the `static_configs_labels` query parameter to set extra static labels. It must contain base64 encoded YAML with labels values.
```

This isn't an endpoint... it's a query param. So it is probably better to add it as it's own sentence vs grouping it with the endpoint description.